lib/ezutils/classes/eztimestamp.php

```diff
@@ -14,6 +14,9 @@ class eZTimestamp
      \return a timestamp in UTC
     */
     public static function getUtcTimestampFromLocalTimestamp( $localTimestamp ) {
+
+        if (!$localTimestamp) return null;
```

Comment:
Not sure if even legacy CS allows brace-less ifs. Same for other change.
```suggestion
        if (!$localTimestamp)
        {
            return null;
        }
```